src/UX/MainMenu/Entry/LinkInterface.php

```diff
@@ -0,0 +1,52 @@
+<?php namespace ILIAS\UX\MainMenu\Entry;
+
+use ILIAS\UX\MainMenu\ChildEntryInterface;
+
+/**
+ * Interface LinkInterface
```

Comment:
Please add a sentece why this is different from the UI link. I know the reasoning, but for other dev's not being involved in the discussion, this might be very hard to grasp.